bootstrap/kubeadm/main.go

```diff
@@ -156,10 +156,6 @@ func main() {
 }
 
 func setupReconcilers(ctx context.Context, mgr ctrl.Manager) {
```

Comment:
Seems like `setupReconcilers` and `setupWebhooks` will need to be consolidated

Reply:
I'm personally for keeping things separated because consolidating could make sense here because there are only one reconciler and few webhook, but in CAPI consolidating most likely is not a good idea because the number of reconciler/webhook is much higher.

Reply:
We should remove them, one of the goals of "moving back to a single manager" is to get back in line with kubebuilder project structure and codebase. A familiar codebase is better for newcomers that might already have experience with a Kubebuilder project

Reply:
TBH I don't think that inlining everything improves readability for newcomers
However, if we want to tackle this lets this refactor let get this in another PR given that this one is already too big

Reply:
can we file an issue for the refactor?

Reply:
https://github.com/kubernetes-sigs/cluster-api/issues/4136